### PR TITLE
readme cleanup

### PR DESCRIPTION
**Pull request by Nikolaj-K**
Added folder tree, syntax, links, shortcuts, prettyification, stratification. 
See
https://github.com/Nikolaj-K/constellation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,90 @@
...
+```
+
+## :computer: Building for development
+### On Mac and Linux
+1. Check out repository 
+```haskell
+git clone git@github.com:Constellation-Labs/constellation.git
+```
+2. From root directory ``constellation, run 
+```haskell
```
Comment (prestonparris) [inline README.md]: I don't think you need Haskell in here? just the back ticks

Comment (Nikolaj-K) [reply README.md]: True, no language annotation required. It should be shell or bash but they don't have shell and when I didn't find the language list for md markup, I didn't want to make 5 commits to test. Haskell is my default syntax highlighting goto when I want to emphasise /.({ and numbers from text and that's why I went for thus substitude. Can be removed or maybe someone knows which shell varient they recognise.

**Comment by buckysballs** [comment]
This is awesome thanks man!

File: README.md (hunk 1 above)
Comment (prestonparris) [reply README.md]: ah gotcha, makes sense

**Comment by tylerprete** [comment]
@Nikolaj-K much appreciated!

**Comment by tylerprete** [comment]
@Nikolaj-K FYI -- you can see the valid language list here: https://help.github.com/articles/creating-and-highlighting-code-blocks/#syntax-highlighting

also you can experiment with how it looks by using "edit" on github, which has a preview. That said, I tried out sh, bash, console -- if what we want is for the ./ to be emphasized, haskell does seem to produce a nice outcome.

**Comment by Nikolaj-K** [comment]
The highlighting thing might be a detail that eventually someone overwrites without checking the history of the discussion anyway. I have no strong feelings about it, just that I like some colors usually.

As I've linked to before in the discord, as far as clean-your-room goes, I think the Lisk folks have a nice github page with pinned repos and coherent names and pics and emojis to get to what you remember/look to quickly:

https://github.com/LiskHQ

Regarding what's in the Constellation readme right now, there's that short section with the install page that someone proposed. This fits there, but generally such lists can also be collected in "awesome" lists as the python Python language prominently has it, and some other open source project adopt it. I.e. next to the main node/consensus repo and maybe a wallet, have an `awesome-constellation` repo with a currated tabled categorized list of all the stuff, split into a dev section and another. Consider as example the CityOfZion one:

https://github.com/CityOfZion/awesome-neo

**Comment by tylerprete** [comment]
@Nikolaj-K we just enabled building forked pull-requests on CircleCI -- can you make a small commit just to trigger the build?

**Comment by Nikolaj-K** [comment]
@tylerprete Done. I suppose I merge once it's not pending anymore and doesn't fail. 
Just check that platform out - the orion frontend is also there so far, right? 
Speaking of which, are all 3 orion related repos still active?
And speaking of that (just wrote Wyatt on discord about that) the naming conventions of repos are somewhat non-existent, huh? small letters and dashes are standard I think, but is looks a bit scrambled there. What's actually the outlook on how to structure dev contributions on the Constellation Labs user on github in the future?

**Comment by Nikolaj-K** [comment]
Another question:

* What's always publicly viewable of the circleci platform/reports?

* How to read those and why do they pop up from the readme push?

```haskell
ERROR PeerToPeer - Unrecognized authenticated UDP message
```

&

```haskell
[info] org.constellation.cluster.MultiNodeOEDAGTest *** ABORTED ***
[info]   java.util.ConcurrentModificationException: Two threads have apparently attempted to run a suite at the same time. This has resulted in both threads attempting to concurrently change the current Informer. Suite class name: org.constellation.cluster.MultiNodeOEDAGTest
```

* Are all existing tests deterministic?

**Pull request merged**